templates/web/guides/models/pins.md.erb

```diff
@@ -396,6 +396,55 @@ pins(:g1).size      # => 3
 pin_groups.size     # => 1
 ~~~
 
+Pin groups can be created using a block format that allows a mix of Symbols, Arrays, and Regexp.
+
+~~~ruby
+add_pin(:ddr_bus1)
+add_pin(:ddr_bus2)
+add_pin(:ddr_bus3)
+add_pin(:ddr_clk)
+add_pin(:tdi)
+add_pin(:tdo)
+add_pin(:tms)
+add_pin(:tck)
+add_pin(:trst)
+add_pin_groups do
```

Comment:
I still don't get what the block is doing in an example like this?

A block is used like this when defining a register for example, because the outer block owns/defines the register and the inner block configures it.
Here, it seems like the outer block is doing nothing and the inner block is creating independent pin groups; subsequently the API looks weird to me.

i.e. why is it not just:

~~~ruby
add_group :ddr_data_and_clk, /^ddr_bus/, :ddr_clk
add_group :jtag, [:tdo, :tdi, :tms, :tck, :trst]
add_group :jtag_wo_trst do
  with :jtag
  without :trst
end
~~~